qemu/tests/cfg/fio_perf.cfg

```diff
@@ -94,10 +94,12 @@
             fio_options = "--rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=${fs_dest}/%s --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --size=512MB --time_based --output=${guest_result_file} &> /dev/null"
             order_list = "Block_size Iodepth Threads BW(MB/S) IOPS Latency(ms) Host_CPU BW/CPU KVM_Exits"
             fs_binary_extra_options = " --thread-pool-size=32"
-            !Host_RHEL.m8:
-                # The below option isn't supported on RHEL8.
+            Host_RHEL.m9:
```

Comment:
This line would need update later. So this is not the best solution.

Reply:
I get your point. 
Change it with !Host_RHEL.m8:  
Thanks.